integration-tests/slack/post-message-test.js

```diff
@@ -0,0 +1,23 @@
+const { postMessage } = require('./post-message')
+
+
+// Post a message to Slack channels
+const postMessageTest = async () => {
```

Comment:
I am not clear what this is testing. Just that `postMessage` doesn't error?

Reply:
Is this for debugging only (not executed in integration test suite)? Some comments would be helpful.

Reply:
`post-message-test.js` is to show `post-message.js` is working. I forgot to say I will remove it in PR before merge.